sotodlib/coords/pmat.py

```diff
@@ -323,9 +340,16 @@ def to_inverse_weights(self, weights_map=None, tod=None, dest=None,
             weights_map = self.to_weights(
                 tod=tod, comps=comps, signal=signal, det_weights=det_weights, cuts=cuts)
 
+        if self.pix_scheme == "healpix":
+            tile_list = self._get_hp_tile_list(weights_map)
+            if None in weights_map: weights_map = hp_utils.untile_healpix_compressed(weights_map, -1)
```

Comment:
Two lines, please.  Or you can make this a one-liner with an idempotent form of the function, `weights_map = hp_utils.as_compressed(weights_map)`.